portfolio/src/main/webapp/index.html

```diff
@@ -30,6 +30,19 @@ <h1>Heidi Johnson's Portfolio</h1>
             <input type="submit" />
           </form>
           <ul id="comments-container"></ul>
+          
+          <label for="number-comments">Number of comments:</label>
+            <select onchange="storeNumberComments()" name="number-comments" id="number-comments">
+              <option value="0">0</option>
```

Comment:
You don't really need to specify the `value="x"` part here. If the value attribute isn't specified in the `<option>` tag, the content is passed as the value instead. Since the values are the same as the content here, you should still get the same result.